### PR TITLE
- remove UI elements. Leave only Typography

### PR DESCRIPTION
**Pull request by VasylHryha**
Close https://github.com/epicmaxco/vuestic-admin/issues/943
- Tables - remove completely
- Statistics - remove completely
- Forms - remove completely
- Cleanup/fixes
- Update routes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: src/components/navbar/components/dropdowns/ProfileDropdown.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
       <template #anchor>
         <span class="profile-dropdown__anchor">
           <slot />
-          <VaIcon :color="colors.primary" :name="isShown ? 'angle_up' : 'angle_down'" class="px-2" />
+          <VaIcon :color="colors.primary" size="large" :name="isShown ? 'expand_more' : 'expand_less'" class="px-2" />
```
Comment (VasylHryha) [inline src/components/navbar/components/dropdowns/ProfileDropdown.vue:7]: Replaced with `material icons`

**Comment by VasylHryha** [comment]
new routes:
image

**Comment by asvae** [comment]
I suggest we stick to original issue. Or let's discuss if you disagree with something.

* Pages were not renamed (preferences, settings).
* Settings and preferences should not go under payments category, but in the root.
* Recover password email - sidebar navigation not needed (too niche).
* Maps are not needed.

Looks close!

**Comment by VasylHryha** [comment]
latest routes:
image

**Comment by VasylHryha** [comment]
latest version:
image

**Pull request merged**